src/location-registry/controllers/controller.py

```diff
@@ -68,13 +66,13 @@ def register_location():
                 try:
                     latitude = float(json_data["latitude"])
                     longitude = float(json_data["longitude"])
-                except: 
+                except:
                     latitude = None
                     longitude = None
             try:
                 road_intensity = int(json_data["roadIntensity"])
             except:
-                road_intensity = None    
+                road_intensity = None
```

Comment:
I am getting a runtime error on this line: "**TypeError: string indices must be integers**"
![image](https://user-images.githubusercontent.com/1590213/110856354-97fde100-82c8-11eb-80ff-2fad61238513.png)

Reply:
could you share the request body? see an example of what works below:
```
{
   "locationReference":"loc_200",
   "hostName":"host_100",
   "mobility": "Static",
   "latitude": 0.32,
   "longitude": 32.10,
   "roadIntensity": "1",
   "description": "ISU",
   "roadStatus": "Paved",
   "localActivities":[{"value":"Burning","label":"Burning"},{"value":"Dust","label":"Dust"},{"value":"Construction","label":"Construction"}]
}
```